src/clj/puppetlabs/services/legacy_routes/legacy_routes_service.clj

```diff
@@ -45,7 +46,8 @@
                                                                    (throw (IllegalStateException.
                                                                             (i18n/trs "Versioned code not supported."))))
                                                                  (constantly nil)
-                                                                 false))
+                                                                 false
+                                                                 boltlib-path))
```

Comment:
Do we care about this config setting for the legacy routes? Can't we just pass nil instead of pulling it out of the config hash?

Reply:
I can update that. I didnt know if that would be needed there...